src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type MoviesArr = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<MoviesArr> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard
+        title={movie.title}
+        description={movie.description}
+        imgUrl={movie.imgUrl}
+        imdbUrl={movie.imdbUrl}
+        imdbId={movie.imdbId}
```

Comment:
You can pass the whole movie and get all needed props inside MovieCard component

Reply:
spread operator?